airflow/providers/amazon/aws/operators/rds.py

```diff
@@ -42,6 +43,15 @@ class RdsBaseOperator(BaseOperator):
     ui_fgcolor = "#ffffff"
 
     def __init__(self, *args, aws_conn_id: str = "aws_conn_id", hook_params: dict | None = None, **kwargs):
+        if hook_params is not None:
+            warnings.warn(
+                "The parameter hook_params is deprecated and will be removed. "
+                "If you were using it, please get in touch either on airflow slack, "
+                "or by opening a github issue on the project. "
```

Comment:
Very informative deprecation message. Just a small suggestion - 
```suggestion
                "or by opening a Github issue on the project. "
```